### PR TITLE
CVSL-1740 Add missing health checks and tidy up integration tests

### PR DESCRIPTION
**Pull request by andrewrlee**
Previously they were all sitting on the same root url meaning you could get clashes and not see where the endpoints were hosted. Now this should make working with them easier

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '3.1'
 services:
 
   redis:
-    image: 'redis:5.0'
+    image: 'redis:7.0'
```
Comment (andrewrlee) [inline docker-compose.yml:5]: bump redis to match prod version

File: feature.env [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
-PROBATION_SEARCH_API_URL=http://localhost:9091
+TOKEN_VERIFICATION_API_URL=http://localhost:9091/verification-api
+LICENCE_API_URL=http://localhost:9091/licences-api
+COMMUNITY_API_URL=http://localhost:9091/community-api
+PRISONER_SEARCH_API_URL=http://localhost:9091/prisoner-search-api
+PRISON_API_URL=http://localhost:9091/prison-api
+PRISON_REGISTER_API_URL=http://localhost:9091/prison-register-api
+PROBATION_SEARCH_API_URL=http://localhost:9091/probation-search-api
 MANAGE_USERS_API_URL=http://localhost:9091/manage-users-api
+COMPONENT_API_URL=http://localhost:9091/components-api
```
Comment (andrewrlee) [inline feature.env:11]: each mock api on own prefix

File: integration_tests/mockApis/feComponent.ts [hunk 3]
```diff
@@ -70,9 +70,21 @@ const stubFeComponentsFail = () =>
...
+const stubFeComponentsPing = () =>
+  stubFor({
+    request: {
+      method: 'GET',
+      urlPattern: '/components-api/ping',
+    },
+    response: {
+      status: 200,
+    },
+  })
```
Comment (andrewrlee) [inline integration_tests/mockApis/feComponent.ts:82]: adding missing ping stubbing

File: server/services/healthCheck.ts [hunk 4]
```diff
@@ -42,19 +42,9 @@ function gatherCheckInfo(aggregateStatus: Record<string, unknown>, currentStatus
...
-          'tokenVerification',
-          `${config.apis.tokenVerification.url}/health/ping`,
-          config.apis.tokenVerification.agent
-        ),
-      ]
-    : []),
-]
+const apiChecks = Object.entries(config.apis).map(([name, api]) =>
+  service(name, `${api.url}${api.healthPath}`, api.agent)
+)
```
Comment (andrewrlee) [inline server/services/healthCheck.ts:47]: automatically register api health endpoints

File: integration_tests/mockApis/community.ts [hunk 5]
```diff
@@ -384,17 +384,14 @@ export default {
     })
   },
 
-  stubGetCutOffDateForLicenceTimeOut: (): SuperAgentRequest => {
```
Comment (louisenorris) [inline integration_tests/mockApis/community.ts:387]: Probably obvious but assume we no longer need this or are stubbing hard stop cut off date elsewhere?

**Review by louisenorris — APPROVED** [review]
LGTM

**Pull request merged**